jetty-client/src/main/java/org/eclipse/jetty/client/HttpDestination.java

```diff
@@ -490,7 +513,127 @@ public String toString()
                 exchanges.size(),
                 connectionPool);
     }
-    
+
+    @FunctionalInterface
+    public interface Multiplexed
+    {
+        void setMaxRequestsPerConnection(int maxRequestsPerConnection);
+    }
+
+    public static class Info
```

Comment:
I like the class.... but not sure `Info` is the right name. `Target`??

Reply:
I chose `Key` - it's the key we use to hold destinations in a `Map`.